src/components/AppNavigationToggle/AppNavigationToggle.vue

```diff
@@ -47,6 +48,9 @@ export default {
 	methods: {
 		toggleNavigation() {
 			this.$emit('update:open', !this.open)
+			emit('navigation-toggled', {
```

Comment:
I would rather have this in the AppNavigation component
https://github.com/nextcloud/nextcloud-vue/blob/89fab6ea8e63592d65519479ac9650cb59343927/src/components/AppNavigation/AppNavigation.vue#L60

Reply:
Fine with me, but then we need a watcher for the open state.